.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: node
+node_js: lts/*
```

Comment:
What's this?

Reply:
it was failing with node v11 which is latest version. doesn't fail on v10 which is lts.

Reply:
all PRs would be failing if i hadn't changed this, because v11 just came out about 10 days ago.